python/ray/dashboard/modules/metrics/dashboards/data_dashboard_panels.py

```diff
@@ -544,6 +544,10 @@
     name="DATA",
     default_uid="rayDataDashboard",
     panels=DATA_GRAFANA_PANELS,
-    standard_global_filters=['dataset=~"$DatasetID"', 'SessionName=~"$SessionName"'],
+    standard_global_filters=[
+        'dataset=~"$DatasetID"',
+        'SessionName=~"$SessionName"',
+        'ray_io_cluster=~"$ray_io_cluster"',
```

Comment:
I'm wondering if we should name the variable to `RayCluster` to make it understandable for people without context of the label.